app/controllers/react_controller.rb

```diff
@@ -1,3 +1,7 @@
 class ReactController < ApplicationController
   layout 'layouts/react_application'
 end
+
+def index
+  render 'react/index'
```

Comment:
Instead of creating an empty view, is it possible to render the `layouts/react_application` without rendering a view inside?
@ohadlevy @tbrisker @ShimShtein?